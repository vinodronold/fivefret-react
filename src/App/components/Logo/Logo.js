// @flow
import React from 'react'

type Props = {
  height: number,
  color: string
}
const _defaultProps = {
  height: 110,
  color: '#222'
}

const Logo = (props: Props) => (
  <svg
    viewBox="0 0 220 110"
    style={{
      margin: 'auto',
      position: 'relative',
      height: props.height,
      width: 'auto'
    }}
  >
    <rect
      x="10"
      y="10"
      width="200"
      height="50"
      rx="4"
      ry="4"
      style={{ fill: 'none', stroke: props.color }}
    />
    <path
      d="M10 20h200M10 30h200M10 40h200M10 50h200M55.6 10v50m37.5-50v50m40-50v50m42.2-50v50"
      style={{ fill: 'none', stroke: props.color }}
    />
    <circle
      cx="113.1"
      cy="35"
      r="1.5"
      style={{ fill: 'none', stroke: props.color }}
    />
    <circle
      cx="191.8"
      cy="35"
      r="1.5"
      style={{ fill: 'none', stroke: props.color }}
    />
    <text
      x="110"
      y="80"
      style={{
        fontFamily: 'Arial',
        fontSize: '1.5rem',
        fill: props.color,
        textAnchor: 'middle'
      }}
    >
      <tspan dy="5.5"> f i v e f r e t s</tspan>
    </text>
  </svg>
)

Logo.defaultProps = _defaultProps
export default Logo
