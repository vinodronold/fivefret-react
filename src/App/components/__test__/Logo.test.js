import React from 'react'
import renderer from 'react-test-renderer'
import Logo from '../Logo'

describe('components/Logo', () => {
  it('should match Snapshot', () => {
    let _Logo = renderer.create(<Logo />)
    expect(_Logo).toMatchSnapshot()
  })
})
