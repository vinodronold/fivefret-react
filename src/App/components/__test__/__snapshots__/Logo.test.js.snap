// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Logo should match Snapshot 1`] = `
<svg
  style={
    Object {
      "height": 110,
      "margin": "auto",
      "position": "relative",
      "width": "auto",
    }
  }
  viewBox="0 0 220 110"
>
  <rect
    height="50"
    rx="4"
    ry="4"
    style={
      Object {
        "fill": "none",
        "stroke": "#222",
      }
    }
    width="200"
    x="10"
    y="10"
  />
  <path
    d="M10 20h200M10 30h200M10 40h200M10 50h200M55.6 10v50m37.5-50v50m40-50v50m42.2-50v50"
    style={
      Object {
        "fill": "none",
        "stroke": "#222",
      }
    }
  />
  <circle
    cx="113.1"
    cy="35"
    r="1.5"
    style={
      Object {
        "fill": "none",
        "stroke": "#222",
      }
    }
  />
  <circle
    cx="191.8"
    cy="35"
    r="1.5"
    style={
      Object {
        "fill": "none",
        "stroke": "#222",
      }
    }
  />
  <text
    style={
      Object {
        "fill": "#222",
        "fontFamily": "Arial",
        "fontSize": "1.5rem",
        "textAnchor": "middle",
      }
    }
    x="110"
    y="80"
  >
    <tspan
      dy="5.5"
    >
       f i v e f r e t s
    </tspan>
  </text>
</svg>
`;
